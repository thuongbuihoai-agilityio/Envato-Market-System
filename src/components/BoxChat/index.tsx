import { memo } from 'react';
import { Box, Image, Heading, Flex, Input, Button } from '@chakra-ui/react';

// Constants
import { IMAGES } from '@app/constants';

// Components
import Message from '@app/components/BoxChat/Message';

// Mocks
import { MESSAGE_TIME } from '@app/mocks';

const BoxChatComponent = (): JSX.Element => (
  <Box w="full" bg="background.body.quaternary" borderRadius="lg">
    <Flex
      direction="row"
      justifyContent="space-between"
      alignItems="center"
      justify="center"
      borderBottom="1px solid"
      borderBottomColor="border.tertiary"
      padding="24px 26px"
    >
      <Heading
        as="h3"
        fontWeight="semibold"
        color="text.primary"
        fontSize="2xl"
        textTransform="capitalize"
      >
        team chat
      </Heading>

      <Flex gap={5}>
        <Image
          src={IMAGES.USER_AVATAR.url}
          alt={IMAGES.USER_AVATAR.alt}
          w="52px"
          h={8}
        />

        <Button
          aria-label="btn-icon-plus"
          bg="secondary.800"
          w={9}
          h={9}
          borderRadius="50%"
        >
          <Image
            src="icons/plus.svg"
            alt={IMAGES.USER_AVATAR.alt}
            w={3.5}
            h={3.5}
          />
        </Button>
      </Flex>
    </Flex>

    <Box padding={{ base: '24px 20px', lg: '38px 35px' }}>
      <Message
        content="Hi, What can i help you with?"
        avatarPosition="before"
        avatar={IMAGES.CHAT_USER_AVATAR.url}
        localeTime={MESSAGE_TIME}
      />

      <Message
        avatar={IMAGES.CHAT_USER_AVATAR.url}
        isImage
        localeTime={MESSAGE_TIME}
      />

      <Flex direction="row-reverse">
        <Message
          content="Hello, I want to know more about your product"
          avatarPosition="after"
          isOwnerMessage
          avatar={IMAGES.CHAT_USER_AVATAR.url}
          localeTime={MESSAGE_TIME}
        />
      </Flex>

      <Message
        content="Sure, I can help you with that"
        avatarPosition="before"
        avatar={IMAGES.CHAT_USER_AVATAR.url}
        localeTime={MESSAGE_TIME}
      />

      <Flex justify="center" align="center">
        <Box
          border="1px solid"
          borderColor="border.secondary"
          p="0 20px"
          mt={5}
          borderRadius="lg"
          w="full"
        >
          <Flex direction="row" alignItems="center">
            <Image
              src={IMAGES.ATTACH.url}
              alt={IMAGES.ATTACH.alt}
              cursor="pointer"
            />

            <Input
              variant="authentication"
              ml={5}
              type="number"
              _dark={{
                border: 'none',
              }}
              sx={{ border: 'none', padding: 0 }}
              placeholder="Type your message here"
            />

            <Image
              src={IMAGES.MICRO.url}
              alt={IMAGES.MICRO.alt}
              cursor="pointer"
            />
          </Flex>
        </Box>
        <Image
          mt={5}
          ml={5}
          src={IMAGES.SEND.url}
          alt={IMAGES.SEND.alt}
          cursor="pointer"
        />
      </Flex>
    </Box>
  </Box>
);

const BoxChat = memo(BoxChatComponent);

export default BoxChat;
