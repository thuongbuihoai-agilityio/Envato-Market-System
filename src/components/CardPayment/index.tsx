import { memo } from 'react';
import { Box, Heading } from '@chakra-ui/react';
import { SubmitHandler } from 'react-hook-form';

// Hooks
import { useForm } from '@app/hooks';

// Components
import { CardBalance } from './CardBalance';
import { UserSelector } from './UserSelector';
import { EnterMoney } from './EnterMoney';

export interface CardPaymentProps {
  balance?: number;
  onSubmit?: SubmitHandler<TTransfer>;
}

export type TTransfer = {
  amount: string;
  userId: string;
};

const CardPaymentComponent = ({
  balance = 24098,
  onSubmit = () => {},
}: CardPaymentProps): JSX.Element => {
  const {
    control,
    handleSubmit,
    formState: { isValid, isSubmitting },
  } = useForm<TTransfer>({});

  return (
    <Box
      p={4}
      w="full"
      bg="background.body.quaternary"
      py={{ base: 4, md: 5 }}
      px={{ base: 4, md: 10 }}
      borderRadius="lg"
    >
      <Heading
        as="h3"
        fontWeight="bold"
        color="text.primary"
        fontSize="lg"
        mb={3}
        textTransform="capitalize"
      >
        my wallet
      </Heading>

      <CardBalance balance={balance} />

      <Box as="form" mt={4} onSubmit={handleSubmit(onSubmit)}>
        <UserSelector control={control} />
        <EnterMoney isDisabled={!isValid || isSubmitting} control={control} />
      </Box>
    </Box>
  );
};

const CardPayment = memo(CardPaymentComponent);

export default CardPayment;
