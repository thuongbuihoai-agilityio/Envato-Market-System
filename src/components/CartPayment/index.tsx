import { memo } from 'react';
import {
  Box,
  Text,
  Image,
  Select,
  Heading,
  Flex,
  Input,
  Button,
} from '@chakra-ui/react';

// Assets
import CARD_PAYMENT from 'src/assets/images/card-payment.svg';
import { ChevronIcon } from '@assets/images/ChevronIcon';

// Constants
import { IMAGES } from '@constants/index';

const CartPaymentComponent = (): JSX.Element => (
  <Box p={4} w="full" bg="background.body.quaternary" padding="20px 42px">
    <Heading
      as="h3"
      fontWeight="bold"
      color="text.primary"
      fontSize="lg"
      mb={3}
      textTransform="capitalize"
    >
      my wallet
    </Heading>

    <Image src={CARD_PAYMENT} alt="Payment Card" />

    <Box mt={4}>
      <Text
        fontWeight="bold"
        color="text.primary"
        fontSize="lg"
        mb={3}
        textTransform="capitalize"
      >
        quick transfer
      </Text>

      <Box position="relative">
        <Select
          size="lg"
          sx={{
            paddingLeft: '50px',
          }}
          borderColor="border.secondary"
          color="text.primary"
          icon={<ChevronIcon />}
        >
          <option value="debit" color="text.primary">
            Debit
          </option>
        </Select>

        <Image
          src={IMAGES.DEBIT_ICON.url}
          alt={IMAGES.DEBIT_ICON.alt}
          boxSize={6}
          position="absolute"
          left={5}
          top="50%"
          transform="translateY(-50%)"
        />

        <Text
          sx={{
            position: 'absolute',
            right: 10,
            top: '50%',
            transform: 'translateY(-50%)',
          }}
          fontWeight="bold"
          fontSize="sm"
        >
          $ 10,431
        </Text>
      </Box>

      <Box
        border="1px solid"
        borderColor="border.secondary"
        p={4}
        mt={5}
        borderRadius="lg"
      >
        <Text color="text.secondary"> Enter amount </Text>
        <Flex direction="row" alignItems="center">
          <Text color="text.primary" fontSize="2xl" fontWeight="bold">
            $
          </Text>
          <Input
            variant="authentication"
            type="number"
            _dark={{
              border: 'none',
            }}
            sx={{ border: 'none', padding: 0 }}
          />

          <Image
            src={IMAGES.USER_AVATAR.url}
            alt={IMAGES.USER_AVATAR.alt}
            boxSize={6}
            mt={3}
            w="42px"
          />
        </Flex>
      </Box>

      <Button mt={4} colorScheme="primary" fontWeight="bold">
        Send Money
      </Button>
    </Box>
  </Box>
);

const CardPayment = memo(CartPaymentComponent);

export default CardPayment;
