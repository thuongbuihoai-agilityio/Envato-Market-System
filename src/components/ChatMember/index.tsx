import { memo } from 'react';
import {
  Box,
  Flex,
  Text,
  Avatar,
  AvatarBadge,
  useColorModeValue,
} from '@chakra-ui/react';

// Themes
import { colors } from '@app/themes/bases/colors';

// Utils
import { getStatusColor } from '@app/utils';

export type Props = {
  avatar?: string;
  name?: string;
  status?: string;
  localeTime?: string;
  icon?: React.ReactNode;
  statusColor?: string;
  onClick?: () => void;
};

const ChatMember = ({
  avatar,
  name,
  localeTime,
  icon,
  statusColor = '',
  onClick,
}: Props) => {
  const colorFill = useColorModeValue(
    colors.secondary[200],
    colors.secondary[600],
  );

  return (
    <Box
      cursor="pointer"
      _hover={{ bg: colorFill }}
      onClick={onClick}
      borderRadius={'lg'}
    >
      <Flex justify="space-between" p={3.5}>
        <Flex gap={3}>
          <Avatar src={avatar} borderRadius="50%">
            <AvatarBadge boxSize={4} bg={getStatusColor(statusColor)} top={7} />
          </Avatar>
          <Box mr={6}>
            <Text>{name}</Text>
          </Box>
        </Flex>

        <Flex direction="column">
          <Text>{localeTime}</Text>
          {icon}
        </Flex>
      </Flex>
    </Box>
  );
};

const ChatMemberMemorized = memo(ChatMember);

export default ChatMemberMemorized;
