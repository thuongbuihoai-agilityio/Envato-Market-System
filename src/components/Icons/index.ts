export * from './Bell';
export * from './Email';
export * from './Gift';
export * from './LightTheme';
export * from './DarkTheme';
export * from './User';
export * from './Logout';
export * from './Arrow';
export * from './Dashboard';
export * from './Transaction';
export * from './MyWallet';
export * from './History';
export * from './Setting';
export * from './SignOut';
export * from './Rotate';
export * from './Search';
export * from './Sort';
export * from './Account';
export * from './Dot';
export * from './Filter';
export * from './Search';
export * from './BookMart';
export * from './ShareNodes';
export * from './AvatarSetting';
export * from './Faq';
export * from './MyWallet';
export * from './Setting';
export * from './ChevronIcon';
export * from './Eye';
export * from './EyeSlash';
