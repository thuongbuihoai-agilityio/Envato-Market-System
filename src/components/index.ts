export { default as InputField } from './common/InputField';
export { default as Logo } from './common/Logo';
export { default as Benefit } from './common/Benefit';
export { default as Button } from './common/Button';
export { default as Divider } from './common/Divider';
export { default as Menu } from './common/Menu';
export { default as IconButton } from './common/IconButton';
export { default as Avatar } from './common/Avatar';
export { default as Dropdown } from './common/Dropdown';
export { default as TotalCard } from './TotalCard';
export { default as Lazy } from './Lazy';
export { default as TotalListSkeleton } from './Skeleton/TotalListSkeleton';
export { default as Sidebar } from './Sidebar';
export { default as Navigation } from './common/Navigation';
export { default as Header } from './common/Header';
export { default as Select } from './common/Select';
export { default as Pagination } from './common/Pagination';
export { default as FilterUser } from './FilterUser';
export { default as Table, type TDataSource } from './common/Table';
export * from './common/Table/Body';
export { default as HeadCell } from './common/Table/HeadCell';
export { default as TotalList } from './TotalList';
export { default as CartPayment } from './CartPayment';
export { default as BoxChat } from './BoxChat';
export { default as Efficiency } from './Efficiency';
export { default as RevenueFlow } from './RevenueFlow';
export { default as OverallBalance } from './OverallBalance';
export { default as TotalBalance } from './TotalBalance';
export { default as ExpandSidebar } from './ExpandSidebar';
export { default as MiniSidebar } from './MiniSidebar';
