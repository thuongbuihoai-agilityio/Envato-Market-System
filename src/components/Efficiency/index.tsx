import { memo, useCallback } from 'react';
import isEqual from 'react-fast-compare';

// Components
import {
  Box,
  Flex,
  Heading,
  Text,
  theme,
  useColorModeValue,
} from '@chakra-ui/react';
import { Select } from '@app/components';
import { TOption } from '@app/components/common/Select';
import EfficiencyInfo from './EfficiencyInfo';
import EfficiencyRefetch from './Refetching';

// Icons
import { Arrow } from '@app/assets/icons';

// Constants
import { EFFICIENCY_OPTIONS } from '@app/constants';

// Types
import { IEfficiency } from '@app/interfaces';

interface EfficiencyProps extends IEfficiency {
  isLoading?: boolean;
  isExchangeRate?: boolean;
  isLoadingWhenSelect?: boolean;
  onChangeSelect: (data: TOption) => void;
}

const EfficiencyComponent = ({
  arrival,
  spending,
  statistical,
  isLoading = false,
  isLoadingWhenSelect = false,
  onChangeSelect,
}: EfficiencyProps) => {
  const colorFill = useColorModeValue(
    theme.colors.gray[400],
    theme.colors.white,
  );

  const renderTitle = useCallback(
    ({ label }: TOption) => (
      <Flex alignItems="center">
        <Text>{label}</Text>
        <Arrow color={colorFill} />
      </Flex>
    ),
    [colorFill],
  );

  return (
    <Box bg="background.component.primary" rounded="lg">
      <Flex
        py={4}
        px={5}
        borderBottom="1px"
        borderColor="border.primary"
        justifyContent="space-between"
      >
        <Heading variant="heading2Xl" as="h3">
          Efficiency
        </Heading>
        <Box w={102} h={21}>
          <Select
            options={EFFICIENCY_OPTIONS}
            size="sm"
            variant="no-background"
            renderTitle={renderTitle}
            onSelect={onChangeSelect}
          />
        </Box>
      </Flex>
      {isLoadingWhenSelect && isLoading ? (
        <EfficiencyRefetch />
      ) : (
        <EfficiencyInfo
          spending={spending}
          statistical={statistical}
          arrival={arrival}
        />
      )}
    </Box>
  );
};

const Efficiency = memo(EfficiencyComponent, isEqual);

export default Efficiency;
