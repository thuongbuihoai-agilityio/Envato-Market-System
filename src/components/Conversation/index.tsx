import { memo, useMemo, useState } from 'react';
import { Box, Heading, Flex, theme, useColorModeValue } from '@chakra-ui/react';
import ReactQuill from 'react-quill';
import 'react-quill/dist/quill.snow.css';

// themes
import { colors } from '@app/themes/bases/colors';

// Constants
import { AVATAR_POSITION, IMAGES } from '@app/constants';

// Components
import { Message } from '@app/components';
import { ChatMemberMemorized } from '@app/components';
import { CustomButton } from '@app/components';
import { SendIcon } from '@app/components/Icons';

// Mocks
import { MESSAGE_TIME, MESSAGES } from '@app/mocks';

// Stores
import { authStore } from '@app/stores';

// Interfaces
import { MessageType } from '@app/interfaces/messages';

export type Props = {
  activeMember?: string;
  filteredMessages: MessageType[];
};

const Conversation = ({ filteredMessages }: Props) => {
  const avatarURL = authStore(
    (state): string | undefined => state.user?.avatarURL,
  );

  const username = authStore(
    ({ user }): string | undefined => `${user?.firstName} ${user?.lastName}`,
  );

  const [editorValue, setEditorValue] = useState<string>('');
  const colorFill = useColorModeValue(
    theme.colors.white,
    colors.secondary[400],
  );

  const messagesToShow = useMemo(
    () => (filteredMessages.length > 0 ? filteredMessages : MESSAGES),
    [filteredMessages],
  );

  return (
    <Box w="full" borderRadius="lg">
      <Flex
        direction="row"
        justifyContent="space-between"
        padding={{ base: '8px', lg: '24px 26px' }}
        bg="background.body.septenary"
      >
        <Heading
          as="h3"
          fontWeight="semibold"
          color="text.primary"
          fontSize="2xl"
          textTransform="capitalize"
        >
          <ChatMemberMemorized
            avatar={avatarURL}
            name={username}
            status="Online"
            statusColor="online"
          />
        </Heading>
      </Flex>

      <Box padding={{ base: '24px 20px', lg: '38px 35px' }}>
        {messagesToShow.map((message): JSX.Element => {
          const { isSend, isAudio, uid, content } = message;

          return (
            <Message
              key={uid}
              content={content}
              isImage={isAudio}
              isOwnerMessage={isSend}
              avatarPosition={
                isSend ? AVATAR_POSITION.AFTER : AVATAR_POSITION.BEFORE
              }
              avatar={IMAGES.CHAT_USER_AVATAR.url}
              localeTime={MESSAGE_TIME}
            />
          );
        })}

        <Flex direction="column" width="full">
          <ReactQuill
            value={editorValue}
            onChange={(value) => setEditorValue(value)}
            modules={{
              toolbar: [
                ['bold', 'italic', 'underline'],
                ['image', 'code-block'],
              ],
            }}
            style={{
              width: '100%',
              backgroundColor: colorFill,
            }}
            theme="snow"
          />

          <CustomButton
            w="unset"
            px={4}
            py={2.5}
            leftIcon={<SendIcon />}
            fontSize="md"
            fontWeight="semibold"
            bgColor="primary.600"
            mt={4}
            alignSelf="flex-end"
          >
            Send
          </CustomButton>
        </Flex>
      </Box>
    </Box>
  );
};

const ConversationMemorized = memo(Conversation);

export default ConversationMemorized;
