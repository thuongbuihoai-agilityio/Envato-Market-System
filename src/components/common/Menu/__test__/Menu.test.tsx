import { screen, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

// components
import { Menu } from '@app/components';

// Assets
import { DashboardIcon } from '@app/components/Icons';
import userEvent from '@testing-library/user-event';

const useNavigateMock = jest.fn();

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => useNavigateMock,
}));

describe('Menu test case', () => {
  const MOCK_ITEM_LIST = [
    {
      id: 1,
      leftIcon: DashboardIcon,
      menuItemContent: 'Dashboards',
      destination: '/',
    },
    {
      id: 2,
      menuItemContent: 'Mock',
      destination: '/mock',
    },
    {
      id: 3,
      menuItemContent: 'Sign Out',
      destination: '/Mock',
    },
  ];

  it('should render correctly and expand by default', () => {
    const { container } = render(
      <MemoryRouter>
        <Menu title="Mock" listItem={MOCK_ITEM_LIST} />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  it('should render correctly with the given minify option', () => {
    const { container } = render(
      <MemoryRouter>
        <Menu title="Mock" isMinify listItem={MOCK_ITEM_LIST} />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  it('should render correctly with the default empty Heading', () => {
    render(
      <MemoryRouter>
        <Menu listItem={MOCK_ITEM_LIST} />
      </MemoryRouter>,
    );

    const expectedHeading = screen.getByRole('heading', { level: 4 });

    expect(expectedHeading.textContent).toMatch('');
  });

  it('handle click menu item', async () => {
    render(
      <MemoryRouter>
        <Menu listItem={MOCK_ITEM_LIST} />
      </MemoryRouter>,
    );

    const singOut = screen.getByText('Sign Out');

    await userEvent.click(singOut);

    expect(useNavigateMock).toHaveBeenCalledWith('/Mock');
  });
});
