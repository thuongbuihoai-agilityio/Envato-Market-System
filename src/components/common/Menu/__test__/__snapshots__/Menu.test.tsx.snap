// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu test case should render correctly and expand by default 1`] = `
<div>
  <div
    class="chakra-stack css-xgszcm"
  >
    <h4
      class="chakra-heading css-1pcjwh9"
    >
      Mock
    </h4>
    <ul
      aria-label="list-icon"
      class="css-19ibvur"
      role="listbox"
    >
      <li
        aria-label="item-icon"
        class="css-0"
        role="list"
      >
        <a
          aria-label="navigate-item"
          class="chakra-link css-kf8s5h"
          href="/"
        >
          <div
            class="css-1lekzkb"
          >
            <div
              class="css-dyap1g"
            >
              <svg
                fill="none"
                height="21"
                viewBox="0 0 18 21"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  class="path-1"
                  d="M0 8.84719C0 7.99027 0.366443 7.17426 1.00691 6.60496L6.34255 1.86217C7.85809 0.515019 10.1419 0.515019 11.6575 1.86217L16.9931 6.60496C17.6336 7.17426 18 7.99027 18 8.84719V17C18 19.2091 16.2091 21 14 21H4C1.79086 21 0 19.2091 0 17V8.84719Z"
                  fill="#1A202C"
                />
                <path
                  class="path-2"
                  d="M5 17C5 14.7909 6.79086 13 9 13C11.2091 13 13 14.7909 13 17V21H5V17Z"
                  fill="#22C55E"
                />
              </svg>
              <p
                class="chakra-text css-1pvloc"
              >
                Dashboards
              </p>
            </div>
          </div>
        </a>
      </li>
      <li
        aria-label="item-icon"
        class="css-0"
        role="list"
      >
        <a
          aria-label="navigate-item"
          class="chakra-link css-1h1hto7"
          href="/mock"
        >
          <div
            class="css-1lekzkb"
          >
            <div
              class="css-dyap1g"
            >
              <p
                class="chakra-text css-1pvloc"
              >
                Mock
              </p>
            </div>
          </div>
        </a>
      </li>
      <li
        aria-label="item-icon"
        class="css-0"
        role="list"
      >
        <a
          aria-label="navigate-item"
          class="chakra-link css-1h1hto7"
          href="/Mock"
        >
          <div
            class="css-1lekzkb"
          >
            <div
              class="css-dyap1g"
            >
              <p
                class="chakra-text css-1pvloc"
              >
                Sign Out
              </p>
            </div>
          </div>
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Menu test case should render correctly with the given minify option 1`] = `
<div>
  <div
    class="chakra-stack css-14jlx9m"
  >
    <ul
      aria-label="list-icon"
      class="css-klyujl"
      role="listbox"
    >
      <li
        aria-label="item-icon"
        class="css-0"
        role="list"
      >
        <a
          aria-label="navigate-item"
          class="chakra-link css-kf8s5h"
          href="/"
        >
          <svg
            fill="none"
            height="21"
            viewBox="0 0 18 21"
            width="18"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="path-1"
              d="M0 8.84719C0 7.99027 0.366443 7.17426 1.00691 6.60496L6.34255 1.86217C7.85809 0.515019 10.1419 0.515019 11.6575 1.86217L16.9931 6.60496C17.6336 7.17426 18 7.99027 18 8.84719V17C18 19.2091 16.2091 21 14 21H4C1.79086 21 0 19.2091 0 17V8.84719Z"
              fill="#1A202C"
            />
            <path
              class="path-2"
              d="M5 17C5 14.7909 6.79086 13 9 13C11.2091 13 13 14.7909 13 17V21H5V17Z"
              fill="#22C55E"
            />
          </svg>
        </a>
      </li>
      <li
        aria-label="item-icon"
        class="css-0"
        role="list"
      >
        <a
          aria-label="navigate-item"
          class="chakra-link css-1h1hto7"
          href="/mock"
        />
      </li>
      <li
        aria-label="item-icon"
        class="css-0"
        role="list"
      >
        <a
          aria-label="navigate-item"
          class="chakra-link css-1h1hto7"
          href="/Mock"
        />
      </li>
    </ul>
  </div>
</div>
`;
